src/main/java/de/cyklon/reflection/entities/impl/OfflinePackageImpl.java

```diff
@@ -0,0 +1,155 @@
+package de.cyklon.reflection.entities.impl;
+
+import de.cyklon.reflection.entities.ClassFile;
+import de.cyklon.reflection.entities.OfflinePackage;
+import de.cyklon.reflection.entities.ReflectPackage;
+import de.cyklon.reflection.exception.NotFoundException;
+import de.cyklon.reflection.function.Filter;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.Unmodifiable;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class OfflinePackageImpl implements OfflinePackage {
+
+	private final String packageName;
+
+	OfflinePackageImpl(String packageName) {
+		this.packageName = packageName;
+	}
+
+	@NotNull
+	@SuppressWarnings("ConstantConditions")
+	public static OfflinePackage get(@NotNull String packageName) {
+		packageName = packageName.strip();
+		if (checkPackage(packageName)) throw new NotFoundException(packageName, "package", "");
+		if (ReflectPackage.BASE_PACKAGE == null) return new OfflinePackageImpl(packageName);
+		return packageName.isBlank() ? ReflectPackage.BASE_PACKAGE : new OfflinePackageImpl(packageName);
+	}
+
+	//returns true if package was not found
```

Comment:
The return value of this method is confusing. It should return true if it is successful and false if it fails